### ISSUE TITLE
Need document for setup GraphEngine cluster

### ISSUE THREAD

**Issue by hcoona** [microsoft/GraphEngine#295]
I'd like to set up a GraphEngine cluster on several Ubuntu machines. Is there any document for it?

**Comment by hcoona** [comment]
The [Getting Started](https://www.graphengine.io/docs/manual/index.html) document only illustrated several concepts. Without concrete instructions, people wouldn't know how to make it run.

**Comment by TaviTruman** [comment]
@hcoona you are correct we don't have any examples on tap that demonstrate how to set up a cluster but it is not very hard to do. I'm not a Ubuntu guy but I will put together the configuration file and or you can use the APIs. We need to build a demo set up app then via PR get it added to the samples dir in the repro.

**Comment by TaviTruman** [comment]
@hcoona I am working on a new sample that demonstrates how to set up GE Availability Group with Server-side and Client-side coding strategies. We are testing with GE App Servers hosted via **Worker Service** running as a **Windows Service** on Windows and as a **Deamon** on Unbuntu. I Should have the demo available in a couple of weeks.

**Comment by edouardpoitras** [comment]
Any progress on this? I'd love to be able to see a working example of a GraphEngine cluster.

**Comment by TaviTruman** [comment]
This has been solved and I have created new documentation and sample code; I will publish it in the Discussion section and provide the link.
